src/garage/torch/modules/tanh_gaussian_mlp_module.py

```diff
@@ -0,0 +1,242 @@
+            to avoid numerical issues (plain value - not log or exponentiated).
+        std_parameterization (str): How the std should be parametrized. There
+            are two options:
+            - exp: the logarithm of the std will be stored, and applied a
+               exponential transformation
+            - softplus: the std will be computed as log(1+exp(x))
+        layer_normalization (bool): Bool for using layer normalization or not.
+
+    """
+
+    def __init__(self,
+                 input_dim,
+                 output_dim,
+                 hidden_sizes=(32, 32),
+                 hidden_nonlinearity=nn.ReLU,
+                 hidden_w_init=nn.init.xavier_uniform_,
+                 hidden_b_init=nn.init.zeros_,
+                 output_nonlinearity=None,
+                 output_w_init=nn.init.xavier_uniform_,
+                 output_b_init=nn.init.zeros_,
+                 init_std=1.0,
+                 min_std=np.exp(-20.),
+                 max_std=np.exp(2.),
+                 std_parameterization='exp',
+                 layer_normalization=False):
+
+        super(TanhGaussianMLPTwoHeadedModule,
+              self).__init__(input_dim=input_dim,
+                             output_dim=output_dim,
+                             hidden_sizes=hidden_sizes,
+                             hidden_nonlinearity=hidden_nonlinearity,
+                             hidden_w_init=hidden_w_init,
+                             hidden_b_init=hidden_b_init,
+                             output_nonlinearity=output_nonlinearity,
+                             output_w_init=output_w_init,
+                             output_b_init=output_b_init,
+                             learn_std=True,
+                             init_std=init_std,
+                             min_std=min_std,
+                             max_std=max_std,
+                             std_parameterization=std_parameterization,
+                             layer_normalization=layer_normalization)
+        self._shared_mean_log_std_network = MultiHeadedMLPModule(
+            n_heads=2,
+            input_dim=self._input_dim,
+            output_dims=self._action_dim,
+            hidden_sizes=self._hidden_sizes,
+            hidden_nonlinearity=self._hidden_nonlinearity,
+            hidden_w_init=self._hidden_w_init,
+            hidden_b_init=self._hidden_b_init,
+            output_nonlinearities=self._output_nonlinearity,
+            output_w_inits=self._output_w_init,
+            output_b_inits=[
+                nn.init.zeros_,
+                lambda x: nn.init.constant_(x, self._init_std.item())
+            ],
+            layer_normalization=self._layer_normalization)
+
+    def _get_mean_and_log_std(self, *inputs):
+        """Get mean and std of Tanh Gaussian distribution given inputs.
+
+        Args:
+            *inputs: Input to the module.
+
+        Returns:
+            tuple:
```

Comment:
you don't need to do `tuple:` -- the docstring style specifically just says to list one per line

return values don't have names.

basically, like this
```
torch.Tensor: Mean of Tanh Gaussian distribution.
torch.Tensor: Variance of Tanh Gaussian distribution.
```

is that the variance, std, or log std?